src/Apps/MyCollection/Routes/EditArtwork/Components/MyCollectionArtworkFormArtistStep.tsx

```diff
@@ -43,3 +139,29 @@ export const MyCollectionArtworkFormArtistStep: React.FC<MyCollectionArtworkForm
     </AppContainer>
   )
 }
+
+export const MyCollectionArtworkFormArtistStepFragmentContainer = createFragmentContainer(
```

Comment:
**issue(non-blocker):** since we updated relay recently, maybe we can already start using hooks here